features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Configuration/components/MaxBackendTps.jsx

```diff
@@ -0,0 +1,186 @@
+ * @export
+ * @param {*} props
+ * @returns
+ */
+export default function MaxBackendTps(props) {
+    const { api, configDispatcher } = props;
+    const classes = useStyles();
+
+    return (
+        <React.Fragment>
+            <Grid item xs={12}>
+                <ExpansionPanel className={classes.expansionPanel}>
+                    <ExpansionPanelSummary expandIcon={<ExpandMoreIcon />}>
+                        <Typography className={classes.subHeading} variant='h6'>
+                            <FormattedMessage
+                                id='Apis.Details.Configuration.Components.MaxBackendTps.maximum.backend.throughput'
+                                defaultMessage='Backend Throughput'
+                            />
+                            <Tooltip
+                                title={
+                                    <FormattedMessage
+                                        id='Apis.Details.Configuration.components.MaxBackendTps.tooltip'
+                                        defaultMessage={'Limits the total number of calls the API Manager is allowed' +
+                                        ' to make to the backend'}
+                                    />
+                                }
+                                aria-label='APISecurity'
+                                placement='right-end'
+                                interactive
+                            >
+                                <HelpOutline className={classes.iconSpace} />
+                            </Tooltip>
+                        </Typography>
+                    </ExpansionPanelSummary>
+                    <ExpansionPanelDetails className={classes.expansionPanelDetails}>
+                        <FormControl component='fieldset'>
+                            <FormLabel component='legend'>Maximum Throughput</FormLabel>
+                            <RadioGroup
+                                aria-label='change-max-TPS'
+                                name={api.maxTps}
+                                value={api.maxTps === null}
+                                onChange={(event) => {
+                                    configDispatcher({
+                                        action: 'maxTps',
+                                        value:
+                                            event.target.value === 'false' ? { production: null, sandbox: null } : null,
+                                    });
+                                }}
+                                row
+                            >
+                                <FormControlLabel
+                                    value
+                                    control={<Radio color='default' />}
+                                    label='Unlimited'
+                                    labelPlacement='end'
+                                />
+                                <FormControlLabel
+                                    value={false}
```

Comment:
You need to give a string value as the label value no a boolean

Reply:
Fixed